<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Auctoin House</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/icon.ico</normaloff>../images/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#282c34;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" colspan="4">
     <widget class="QFrame" name="topFrame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#282c34;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="whiteMark">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	border-radius: 7px;
	background-color: #ABB2BF;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="whiteLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(171, 178, 191);</string>
       </property>
       <property name="text">
        <string>Unsubscribed item</string>
       </property>
      </widget>
      <widget class="QLabel" name="greenLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #98c379;</string>
       </property>
       <property name="text">
        <string>Subscribed item</string>
       </property>
      </widget>
      <widget class="QLabel" name="greenMark">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	border-radius: 7px;
	background-color: #98c379;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="goldMark">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	border-radius: 7px;
	background-color: #e5c07b;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="goldLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #e5c07b;</string>
       </property>
       <property name="text">
        <string>You have the highest bid!</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="itemId">
       <property name="geometry">
        <rect>
         <x>218</x>
         <y>20</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 2px solid #61AFEF;
	border-radius: 10px;
	background-color: #FFFFFF;
}</string>
       </property>
       <property name="maxLength">
        <number>9</number>
       </property>
       <property name="placeholderText">
        <string> Item ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="bidValue">
       <property name="geometry">
        <rect>
         <x>308</x>
         <y>20</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 2px solid #61AFEF;
	border-radius: 10px;
	background-color: #FFFFFF;
}</string>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
       <property name="placeholderText">
        <string>R$ 0,00</string>
       </property>
      </widget>
      <widget class="QPushButton" name="bidButton">
       <property name="geometry">
        <rect>
         <x>468</x>
         <y>10</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Make a bid</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 4px solid #282c34;
    border-radius: 24px;
	border-style: outset;
    padding: 5px;
}

QPushButton:pressed {
    border-style: inset;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../images/bid.png</normaloff>../images/bid.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="registerButton">
       <property name="geometry">
        <rect>
         <x>528</x>
         <y>10</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Register an item</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 4px solid #282c34;
    border-radius: 24px;
	border-style: outset;
    padding: 5px;
}

QPushButton:pressed {
    border-style: inset;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../images/register.png</normaloff>../images/register.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="listButton">
       <property name="geometry">
        <rect>
         <x>588</x>
         <y>10</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Update item list</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 4px solid #282c34;
    border-radius: 24px;
	border-style: outset;
    padding: 5px;
}

QPushButton:pressed {
    border-style: inset;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../images/list.png</normaloff>../images/list.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QFrame" name="listFrame">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QListWidget" name="itemList">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget {
	border: 1px solid #282C34;
	background-color: rgb(40, 44, 52);
}

QListWidget::item:selected{
	background-color: #434957;
}

QListWidget::item:hover {
	background-color: #434957;
}</string>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollBar" name="itemScrollbar">
         <property name="styleSheet">
          <string notr="true">QScrollBar:vertical {
	border: none;
    background: transparent;
	width: 6px
}

QScrollBar::handle:vertical {
    background: rgb(41, 121, 255);
	border-radius: 3px;
    min-height: 10px;
}

QScrollBar::add-line:vertical {
	border: none;
    background: none;
}

QScrollBar::sub-line:vertical {
    border: none;
    background: none;
}

QScrollBar::add-page:vertical {
	background-color: rgb(40, 44, 52);
}

QScrollBar::sub-page:vertical {
	background-color: rgb(40, 44, 52);
}</string>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2" colspan="3">
     <widget class="QFrame" name="notificationFrame">
      <property name="styleSheet">
       <string notr="true">background-color: #ABB2BF;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTextEdit" name="notifications">
         <property name="styleSheet">
          <string notr="true">QTextEdit {
	border: 1px solid #ABB2BF;
	background-color: #ABB2BF;
}</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollBar" name="scrollbar">
         <property name="styleSheet">
          <string notr="true">QScrollBar:vertical {
	border: none;
    background: transparent;
	width: 6px
}

QScrollBar::handle:vertical {
    background: rgb(146, 151, 162);
	border-radius: 3px;
    min-height: 10px;
}

QScrollBar::add-line:vertical {
	border: none;
    background: none;
}

QScrollBar::sub-line:vertical {
    border: none;
    background: none;
}

QScrollBar::add-page:vertical {
	background-color: transparent;
}

QScrollBar::sub-page:vertical {
	background-color: transparent;
}</string>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
